src/components/global/Navbar.jsx

```diff
@@ -1,8 +1,147 @@
-const Navbar = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+import {
+  AppBar,
+  Box,
+  Hidden,
+  IconButton,
+  List,
+  ListItem,
+  Popover,
+  Toolbar,
+  Typography
+} from '@mui/material';
+import MenuIcon from '@mui/icons-material/Menu';
+import { Brightness4, Brightness7 } from '@mui/icons-material';
+import { BrowserRouter as Route, Link } from 'react-router-dom';
+import { useState, useRef } from 'react';
+
+const logoBlobStyle = {
+  position: 'absolute',
+  bottom: 0,
+  left: -30,
+  right: 0,
+  top: -50,
+  height: '250px',
+  backgroundImage: { md: 'url(/assets/logoBlobNav.svg)', sm: 'none' },
+  backgroundRepeat: 'no-repeat',
+  display: {
+    lg: 'flex',
```

Comment:
That line didn't do anything anymore, forgot to delete 
```
  display: {
    lg: 'flex',
    xs: 'none'
  }
 ```

Reply:
Oh right... so let's clean it up a little more then.  How about instead of rendering a `backgroundImage` based on screen size - we render the `Box` holding it that way?   Which I just noticed could be self-closing too.

`<Box sx={logoBlobStyle} display={{ xs: 'none', md: 'block' }} />`

Reply:
changed